libcomposefs/lcfs-writer.c

```diff
@@ -590,7 +635,7 @@ static int read_content(int fd, size_t size, uint8_t *buf)
 	return 0;
 }
 
-static void digest_to_path(const uint8_t *csum, char *buf)
+void lcfs_digest_to_path(const uint8_t *csum, char *buf)
```

Comment:
This is maybe not a great function to make public it is very easy to use to shoot yourself in the foot, as it requires the caller to allocate space and whatnot. I think its better to move to lcfs-utils.h or something so that it can be used in mkcomposefs without needing to be in the public library ABI forever.

Reply:
With the latest change, this remains as an internal function.